<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recognition</class>
 <widget class="QWidget" name="Recognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Evaluate fingerprint recognition system</string>
   </property>
   <widget class="QPushButton" name="pushButton_Evaluate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start to match</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Threshold">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Match Name:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Draw">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw a ROC curve</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>491</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Matched fingerprints</string>
   </property>
   <widget class="QTextEdit" name="textEdit_Results">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>471</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Enrol and match fingerprint</string>
   </property>
   <widget class="QPushButton" name="pushButton_Enroll">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enrol a fingerprint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__Match">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare fingerpirnts</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>881</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>ROC curve</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
